apps/betterangels-backend/clients/schema.py

```diff
@@ -32,31 +32,74 @@
     UpdateClientProfileInput,
 )
 
-CLIENT_RELATED_CLS_NAME_BY_RELATED_NAME = {
-    "contacts": "ClientContact",
-    "hmis_profiles": "HmisProfile",
-    "household_members": "ClientHouseholdMember",
-    "social_media_profiles": "SocialMediaProfile",
+CLIENT_RELATED_CLS_NAME_BY_RELATED_NAME: dict[str, dict[str, Any]] = {
```

Comment:
this dict can be simplified by setting some default values in the mutation. e.g.

```python
    related_cls_map["module"],
    related_cls_map["cls_name"],
    related_cls_map["generic_relation"],
```
-> 
```python
    related_cls_map.get("module", "clients"),
    related_cls_map["cls_name"],
    related_cls_map.get("generic_relation", "False"),
```

Reply:
Isn't there a way to directly get the cls from the relation?